_rules/inline-link-in-paragraph-has-distinguishable-box-shadow-66e9f0.md

```diff
@@ -0,0 +1,138 @@
+---
+id: 66e9f0
+name: Inline link has distinguishable box-shadow
+rule_type: atomic
+description: |
+  This rule checks that inline links have a box-shadow that distinguishes them from the surrounding text
+accessibility_requirements:
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Brian Bors
+    - Carlos Duarte
+  previous authors:
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any [visible][] HTML element that is a [semantic link][], where the closest element (including itself) that generates an [inline box][] is part of a [line box][] that has [inline boxes][inline box] generated from an element containing [visible][] [text nodes][text node] that do not have a [semantic link][] as a [descendant][] in the [flat tree][].
+
+**Note:** The test targets are visible link elements that are part of a block of content that is displayed inline.
+
+## Expectation
+
+Each target element has a `box-shadow` style property with a [computed](https://drafts.csswg.org/css-cascade/#computed-value) [color value](https://drafts.csswg.org/css-backgrounds/#shadow-color) that makes it [visible][].
```

Comment:
Why not do the same thing you did with the border rule, just say "visible box-shadow".